ui/apps/platform/src/Containers/Clusters/ClustersTablePanel.tsx

```diff
@@ -403,6 +408,17 @@ function ClustersTablePanel({
                                     </Button>
                                 </ToolbarItem>
                             )}
+                            {hasAdminRole && (
```

Comment:
Wow, I forgot that we (who is me) added `hasAdminRole` for **Init bundles** for conditional rendering.

Questions outside the scope of this contribution, for your mental slow cooker:
* What are risk and reward to add **Admin** user role as requirement to render the routes:
    https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/routePaths.ts#L213-L220
* Can you verify with install team whether `'Integration'` resource is indeed required for CRS?